src/wp-includes/blocks.php

```diff
@@ -858,7 +858,7 @@ function make_after_block_visitor( $hooked_blocks, $context ) {
 	 * @param array $next         The next sibling block of the given block.
 	 * @return string The serialized markup for the given block, with the markup for any hooked blocks appended to it.
 	 */
-	return function ( &$block, $parent_block = null, $next = null ) use ( $hooked_blocks, $context ) {
+	return function ( &$block, &$parent_block = null, $next = null ) use ( $hooked_blocks, $context ) {
```

Comment:
Same here https://github.com/WordPress/wordpress-develop/pull/5598/files#r1377997304.

But also while here, the default value should also be documented using the format of:

```php
Default null.
```

which can be added to the end of each `@param` description, unless too long, then put on the next line.

Reply:
Addressed in https://github.com/WordPress/wordpress-develop/commit/50a340ab06543379ed062ca526d3f0b4e5d99a6d and https://github.com/WordPress/wordpress-develop/pull/5598/commits/39c59938ddb50c8e77be7ec14436a1fc7e5eb578.